packages/api-clients/open-api/m2mGatewayApi.yml

```
@@ -4402,6 +4402,111 @@ paths:
               $ref: "#/components/headers/RateLimitRemainingHeader"
             X-Rate-Limit-Interval:
               $ref: "#/components/headers/RateLimitIntervalHeader"
+  /reversePurposes:
+    post:
+      tags:
+        - purposes
+      summary: Create a new e-service Purpose
+      description: Create a new purpose for an e-service in receive mode
+      operationId: createReversePurpose
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/EServicePurposeSeed"
+        required: true
+      responses:
+        "201":
+          description: EService Purpose created
```

same for consistency
```suggestion
          description: E-Service Purpose created
```